src/sonic-config-engine/openconfig_acl.py

```diff
@@ -18,7 +18,9 @@
 from pyangbind.lib.base import PybindBase
 from decimal import Decimal
 from bitarray import bitarray
-import __builtin__
```

Comment:
As the first line commented, wondering you manually modified it or auto generated it?

Reply:
The whole PR is auto generated except for setup.py file.

Reply:
This is auto generated using 2to3 tool